pandas/tests/indexes/test_multi.py

```diff
@@ -343,6 +346,19 @@ def test_set_levels_labels_names_bad_input(self):
         with tm.assertRaisesRegexp(TypeError, 'string'):
             self.index.set_names(names, level=0)
 
+    def test_series_index(self):
+        # GH14730
+        index = MultiIndex.from_product([[1, 2, 3], ['A', 'B', 'C']])
+        x = Series(index=index, data=range(9))
+        y = Series([1, 3])
```

Comment:
put this is tests/test_multilevel or test_indexing

find a very similar test (iow indexing a mi with list/ndarray) and put after